label-external-pull-request/action.yml

```diff
@@ -0,0 +1,38 @@
+name: Label External Pull Requests
+description: Checks if a PR is a fork and applies an external label accordingly
+
+inputs:
+  github_token:
+    description: The GitHub token to use read member status
+    required: true
+  team:
+    description: The team slug for external collaborators
+    required: true
+  org:
```

Comment:
More a thought than suggestion, should we use something more descriptive for this variable? Maybe `github_organization`?

Reply:
Same goes for team and username